corehq/warehouse/transforms/sql/user_group_dim.sql

```diff
@@ -0,0 +1,43 @@
+INSERT INTO {{ user_group_dim }} (
+    domain,
+    user_dim_id,
+    group_dim_id,
+    dim_last_modified,
+    dim_created_on,
+    deleted
+)
+
+SELECT
+    ud.domain,
+    ud.id,
+    gd.id,
+    now(),
+    now(),
+    combined_user_group_table.deleted
+FROM
+(
+(
+    SELECT
+        UNNEST(user_ids) AS user_id,
+        group_id AS group_id,
+        false AS deleted
+    FROM
+        {{ group_staging }}
+)
+
+UNION
+
+(
+    SELECT
+        UNNEST(removed_user_ids) AS removed_user_id,
```

Comment:
it could be, union just matches on index and then takes the alias of the first query:

![screen shot 2017-06-26 at 4 11 28 pm](https://user-images.githubusercontent.com/918514/27558139-3ac73ba4-5a8a-11e7-8950-25138e49cc7a.png)

it was just a semantic name